Sources/Build/Command.swift

```diff
@@ -29,12 +29,16 @@ struct Target {
     /// to a client wanting to control the build.
     let name: String
 
-    /// A list of commands to run when building the target.  A command may be
+    /// A list of outputs that represent the target.
+    var outputs: [String]
```

Comment:
It's not the case for the Swift targets, which now only depend on the `.swiftmodule` file instead of all all `.swift.o` files. It allowed a lot of the simplification we can see in the SwiftPM manifest.